src/mesido/pycml/component_library/milp/heat/airco.py

```diff
@@ -0,0 +1,53 @@
+from mesido.pycml import Variable
+
+from ._non_storage_component import _NonStorageComponent
+
+
+class Airco(_NonStorageComponent):
+    """
+    The airco component is there to extract thermal power (Heat) out of the network. Typically,
+    this component is used to model aggregated demands.
```

Comment:
Maybe state something like:
aggregated demands or or cold producers (drycoolers).

Otherwise one may not now what the difference with a heat demand might be.